### ISSUE TITLE
Longbow AVRiL

### ISSUE THREAD

**Issue by HellDragon-HK** [GreatEmerald/UT3Weapons#15]
- ANIMATION RELATED
  - [ ] No character fire animation which can go into idle while firing and interfere with aim

 - GAME PLAY RELATED
  - [ ] When locking onto Hellfire SPMA shots are directed toward the underside of the SPMA and hit the ground instead of it doing nothing, unsure if this is true with other ground vehicles

 - SOUND RELATED
  - [X] reload sound after firing in UT2004 is very loud, louder than the shot itself and much louder than the reload in UT3

 - VISUAL RELATED
  - [ ] Projectile comes from character's chest and not the gun (could just be my character model causing that as she holds her gun further to the side than normal UT2004 models do)

  - [ ] Impact on a surface in UT2004 causes a large quick explosion, in UT3 it impacts, explodes and then explodes outward
  - [ ] Projectile is stock UT2004 AVRiL rocket and not the UT3 AVRiL rocket
  - [x] Alt fire doesn't have the targeting laser which can be used without a vehicle in sight even
  - [x] Reload animation in 1st person is odd as we have a rocket that just floats from nowhere into the weapon....uh lol on UT3 on the last shot you don't see the animation but the rocket just appears in the barrel all of a sudden

**Comment by GreatEmerald** [comment]
The targeting laser is there when there is an enemy to lock on to. There is very little point in it when nothing is locking on.

The load animation only looks weird because there's no character hand. In fact the UT2004 AVRiL has the same effect. Can't do anything about that because UT2004 characters have completely different hands.

**Comment by HellDragon-HK** [comment]
You might not like hearing it but Ballistic has hands that hold the weapons and might try looking at that maybe

**Comment by GreatEmerald** [comment]
Do the hands actually correspond to the character you're using?

**Comment by HellDragon-HK** [comment]
That would be a no but they are team colored, still there are a lot of animation types, you wouldn't be able to use the animation and change the mesh?

**Comment by GreatEmerald** [comment]
No. Third-person and first-person meshes are completely different.
And like I said, the stock AVRiL also looks weird because there are no hands. I'd rather keep it as-is, it's really not worth it.

**Comment by HellDragon-HK** [comment]
The hands I was talking about is 1st person only, it has no effect on 3rd if you misunderstood me.

**Comment by GreatEmerald** [comment]
No, I understood you. I'm saying that I can't use third-person hands in first person, and it's not worth porting first person UT3 hands because they won't match the characters like they do in UT3 (and it would be waaay complicated).

**Comment by HellDragon-HK** [comment]
Alright then, later down the line something should probably be done though because a floating rocket reload kind of mars the mod quality in my opinion, even if the hands don't match the character it would be better than floating I would think (think how custom player models in UT3 reuse the same hands for example even if they are nowhere near correct)

**Comment by HellDragon-HK** [comment]
In UT3 AVRiL can target empty vehicles but not in UT2004

In UT3 AVRiL causes little knockback to Scorpion only lifting off the ground slightly, in UT2004 it's causing extreme knockback and flipping it, damage is consistent in both games at 200 damage although in UT3 Scorpion has already lost at least 20 hp from fire burning damage before I can get to it so I tried it with a Goliath and got 200

EDIT - Front hit on a Hellbender causes a lot of knockback, way more than in UT3 which is between little and fair

EDIT2 - Hitting the side of the front causes some serious knockback, in UT3 it's only doing around fair knockback

Hellfire is getting little to fair knockback, in UT3 nothing happens except damage

**Comment by HellDragon-HK** [comment]
On network game AVRiL alt fire lock on is hard to use and takes several seconds to even pickup a target, alt fire is also impossible to use after primary fire until the missile hits or is destroyed

**Comment by HellDragon-HK** [comment]
3rd person reload animation resets per mouse click, meaning if you try to fire before it's fully reloaded the animation starts over, only happens in 3rd and doesn't impact fire rate as a shot can still be fired if the repeating animation isn't finished.

Also there is no character fire animation being used, Rifle nor BigGun so if standing still long enough you can fire shots while in the Idle animation with the gun not even pointed in the right direction.  Likewise if standing still while firing you'll eventually go into Idle animation while still shooting.

**Comment by HellDragon-HK** [comment]
Another new issue, rockets fired at Hellfire SPMA while locked on are directed under it instead of directly at it
